### PR TITLE
Rubicon adapter video support

### PR DESCRIPTION
**Pull request by dbridges12**

### CODE REVIEW

**Comment by dbemiller** [comment]
Don't forget to update the json-schema in `static/bidder-params` ^^

**Comment by dbridges12** [comment]
@dbemiller can you provide some insight as to updating this file please - I'm not seeing much in the other adapter bidder_params files.

**Comment by dbemiller** [comment]
json-schema is an open standard. The high level question it answers is: "how can we rigorously describe which JSON payloads are valid for some use?" json-schemas are JSON blobs which can be used to validate JSON payloads.

The _official_ example page is here: http://json-schema.org/example1.html 
The guide I find most useful is here: https://spacetelescope.github.io/understanding-json-schema/
You can write a json-schema and check various payloads against it here: https://www.jsonschemavalidator.net/

Since you're adding a video param to your adapter, you'll want to update your json-schema to keep them in sync.

The OpenRTB endpoint [actually uses a library](https://github.com/prebid/prebid-server/pull/196/files#diff-87f6b0e44fcf81456d287a66904fd6aeR68) to [validate incoming HTTP request payloads](https://github.com/prebid/prebid-server/pull/196/files#diff-09e5d415f33d74fd6df20b2584f318d1R205). By adding more validation to your json-schema, publishers who miss required params, or send in params of the wrong type, will get nice error messages back about it.

File: adapters/rubicon/rubicon.go [hunk 1]
```diff
@@ -274,8 +295,10 @@ func (a *RubiconAdapter) callOne(ctx context.Context, req *pbs.PBSRequest, reqJS
 
 func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.PBSBidder) (pbs.PBSBidSlice, error) {
 	requests := make([]bytes.Buffer, len(bidder.AdUnits))
+	supportedMediaTypes := []pbs.MediaType{pbs.MEDIA_TYPE_BANNER, pbs.MEDIA_TYPE_VIDEO}
+
 	for i, unit := range bidder.AdUnits {
-		rubiReq, err := adapters.MakeOpenRTBGeneric(req, bidder, a.FamilyName(), []pbs.MediaType{pbs.MEDIA_TYPE_BANNER}, true)
+		rubiReq, err := adapters.MakeOpenRTBGeneric(req, bidder, a.FamilyName(), supportedMediaTypes, true)
```
Comment (cindyhere33) [inline adapters/rubicon/rubicon.go]: Is the last parameter still always true?

**Comment by dbridges12** [comment]
Thanks for the thorough response @dbemiller - I'll have a look at your refs and associated PR.

File: adapters/rubicon/rubicon.go (hunk 1 above)
Comment (dbridges12) [reply adapters/rubicon/rubicon.go]: I believe so since we only support a single media type for a bid request.

**Comment by bretg** [comment]
Great - thanks for the code and reviews. Merging.

**Pull request merged**